src/markdown-helpers.ts

```diff
@@ -320,7 +320,7 @@ export const rawTypeToTypeInformation = (
             }))
           : [],
     };
-  } else if (typeString === 'String') {
+  } else if (typeString === 'String' || typeString === 'string') {
     return {
       collection,
       type: 'String',
```

Comment:
question: I'm not familiar with this code, but does this end up returning the `String` wrapper type anyways?

Reply:
good point. `@electron/typescript-definitions` seems to be mapping `String` to `string` however somewhere.

Reply:
https://github.com/electron/electron/pull/31752#issuecomment-964499950